overlord/snapstate/backend/link.go

```diff
@@ -169,7 +170,6 @@ func (b Backend) generateWrappers(s *snap.Info, linkCtx LinkContext) error {
 		}
 	}()
 
-	disabledSvcs := linkCtx.PrevDisabledServices
```

Comment:
the field can also be removed from the struct I suppose

Reply:
Indeed, thanks, done.